src/pages/Offer/components/StudentOfferBadge.tsx

```diff
@@ -0,0 +1,34 @@
+import { colors } from '@hedviginsurance/brand'
+import { TranslationsConsumer } from '@hedviginsurance/textkeyfy'
+import * as React from 'react'
+import styled from 'react-emotion'
+
+const StudentBadgeWrapper = styled('div')(
+  ({ placement }: { placement: 'left' | 'right' }) => ({
+    position: 'absolute',
+    [placement === 'left' ? 'right' : 'left']: 'calc(100% + 60px)',
+    top: -25,
+    padding: 20,
+    width: 140,
+    height: 140,
+    textAlign: 'center',
+    fontWeight: 'bold',
+    backgroundColor: colors.PINK,
+    color: colors.WHITE,
+    borderRadius: '50%',
+
+    '@media (max-width: 560px)': {
+      display: 'none', // TODO fixme (this is a quick hack to get it working on desktop but not break mobile completely)
```

Comment:
Something something no comments :wink:

No but when do we intend to fix this? 🙂

Reply:
Maybe we should discuss mobile on web onboarding as there are no designs for it, and no traffic is directed to it.

Reply:
Haha if you have a suggestion on how to express this in code that is also easy to find if we're looking for todo/fixme comments i'm all ears ;)

On a more serious note i think its more about "if" over "when", i think it depends on how the web onboarding performs.